contracts/VM.sol

```diff
@@ -89,9 +80,8 @@ abstract contract VM {
                 }(
                     // inputs
                     state.buildInputs(
-                        //selector
                         bytes4(command),
```

Comment:
If a fallback is always signified by a "0" selector then that what require deliberate effort in setting that by the constructing entity. But a fallback is invoked when a function call is mal-formed.
- Could this be extended so that "selector" that doesn't match an existing selector would point to the fallback action to be consistent with the current fallback model?